crates/youki/src/logger.rs

```diff
@@ -93,3 +100,31 @@ where
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::env;
+
+    #[test]
+    fn test_detect_log_level_is_debug() {
+        env::set_var(LOG_LEVEL_ENV_NAME, "error");
```

Comment:
We may have to serialized this. cargo test runs in a single process and env variables are shared across the same process and test cases.

Reply:
Sounds good. I will try to fix it.